CourseApp/CourseApp/CourseApp.cpp

```diff
@@ -0,0 +1,63 @@
+#include <iostream>
+#include <math.h>
+#include <vector>
+
+double func(const double x)
+{
+    if (abs(x) < 1)
+        return acos(x);
+
+    return pow(1.2, x) - pow(x, 1.2);
+}
+
+std::vector<double> exampleA(const double xStart, const double xEnd, const double xDelta)
+{
+    std::vector<double> results;
+
+    for (double x = xStart; x <= xEnd; x += xDelta)
+    {
+        double result = func(x);
+        results.push_back(result);
+    }
+
+    return results;
+}
+
+std::vector<double> exampleB(const double *values, const int size)
+{
+    std::vector<double> results;
+
+    for (int i = 0; i < size; i++)
```

Comment:
Can we iterate over vector without using indexes?

Reply:
for (type item : vector) {}